### ISSUE TITLE
User Navigation and Workflow Needs to be Made Explicit

### ISSUE THREAD

**Issue by billyhunt** [caneruguz/osf-style#13]
We need consistent rules for user navigation and workflow.

Example
-------------

Keep users in the same location unless they specifically navigate to it.  (e.g.  When a user creates a new project, the project is displayed in the project organizer with an open folder and the user remains in the project organizer.)

**Comment by caneruguz** [comment]
This is a good workflow point and I think it's good to place it within the principles section as a general guideline. I'm adding discuss label to see what other's think.

**Comment by haoyuchen1992** [comment]
Yes, After creating a project, it is not necessary for users to edit the project instantly. I also think it is better to remain user in the same page after they create a new project. And then once they want to edit something in new project, they could go into the project.  This workflow follows one of user interface design rules -- always do what users expect.
